### DISCUSSION TITLE
Avestrade there trying to scam me

### DISCUSSION THREAD

**Discussion by Tacety** [blockscout/blockscout#8635] [Support]
I have made an investment with Avestrade like 8 months ago for $340.00 cad  Markus Lavinskie was my finacial adviser my broker he gave me all his cradentals and i payed from my bank to my shakepay than to the company Avestrade so i have all my reciepts and screen shots of all my accounts and we traded for a few months in those 3.5 months we had made aprox $3400.00 i waited anouther .8 of a month so like just over 3/4 till the next month and than i had an familliy emergency so i asked my broker Markus Lavinski for and advance on  my account a withdraw of some of my funds at first he said yes sure that shouldnt be a problem so i waited for a coulpe weeks and nothing so i tryed to ask again and by that point i had advanced to aprox 4500 on my account but when i tryed to call him his number was out of service so than i tryed to contact avestrade and they said i wasnt in there system .when i signed up i actualy signed up on an imperial oil advertizmenrmt and when i pushed the button on imperial oils website it sent me to Avestrade where i linked up with one of there secretarys and than was intercomed to Markus Lavinski  with Avestrade i asked Avestrade Marcus if they were lonked to Imperial Oil and they answerd yes they were and branch off of Imperial Oil And i said cool i just saw an advertizing with Imperial Oil That said on there statment that i am elageble for to make $600.00 a day as a passive income thats why i had signed up there statment or catch phrase at the beginning of there add stated that anyone could make $600.00 a day if they signed up with Imperial Oil so i did push Iperial  Oils  add button where it said regester my credentials than got imideatly transfered in one click to Avestrade so i payed my $340.00 canadian dollers to Avestrade under the imprestion i was and still should be regestered at Imperial Oil Company Globle  inc.so my question is how can i gain accsess to all of my money and who can help me with this situation that Markus Lavinski put me in plus i did phone at one point Imperial Oil Inc And they did at first verify i had an account with the i was so happy at that point but as time went by sadly i seem to have bee wiped off off there computer data right now im not sure if i have an accont with imperial oil any more i should cause i never closed my accoun down what puzzeled me right from the begiming thow was how come i only made $3400.00 in 3.5 month when i was saposto be making $600.00 a day im very confused of how they think they can get away with this wrongfull and very unprofestional way of doing buisness with me Tyler Autyy Deneault 02/18/1976 tdogjake76@gmail.com 12505742086 Chase BC Canada V0E1M0 5652 KAMLOOPS SHUSWAP RD (CEL7WET RD) PO BOX 1810 CHASE BC V0E1M0 CANADA BCpls help me with some answeres and some protacals what i can do or if u know the best Lawyer for this case thanku Tyler Autry Deneault mon oct 16 2023 pls contact me using my credentials i have provided in my matter at hand thanku

**Comment by Mojmir-R** [comment]
Hey, this seems to be absolutely not related to Blockscout. Please consider deleting your off-topic message. Thank you
